en/guides/using-open-source-software.md

```diff
@@ -65,30 +65,27 @@ As per any software, you should maintain best practices and have a process in pl
 
 Whenever the Crown obtains software under an open source licence, departments should review the terms and conditions to validate if they can accept and comply with them given their particular business context.
 
-The software is usually provided 'as-is' meaning the community will not accept liability or provide any financial compensation to the Crown for service interruption, loss of data, or loss of confidentiality. Data ownership and processing may remain with the user.
+The software is usually provided 'as-is' meaning the community will not accept liability or provide any financial compensation to the Crown for service interruption, loss of data, or loss of confidentiality.
 
-All software licensed under an [Open Source Initiative approved license](https://opensource.org/licenses/alphabetical) or a [Free Software Foundation free software licence](https://www.gnu.org/licenses/license-list.html#SoftwareLicenses) is considered open source software and can be used by GC [without modifications](#use-open-source-software-without-modification).
+All software licensed under an [Open Source Initiative approved license](https://opensource.org/licenses/alphabetical) or a [Free Software Foundation free software licence](https://www.gnu.org/licenses/license-list.html#SoftwareLicenses) is considered open source software and can be used by the GC [without modifications](#use-open-source-software-without-modification).
 
 However if the software needs to be modified, the following considerations should be applied to help identify what licences terms and conditions the department is willing to approve.
 
 <!-- markdownlint-disable MD006 -->
 <!-- markdownlint-disable MD029 -->
-1. Will you need to be making modifications to the source code of the application during its life cycle or will the software be used as a component of a custom development project?
```

Comment:
The use of the OSS as a component a of a development is important, not from a release point of view but because of the choice of licence you will be applying on it. So should point to Publishing instead if we remove here?

Reply:
The sentence right above
> if the software needs to be modified, the following considerations should be applied to help identify what licences terms and conditions the department is willing to approve

You only move into the decision tree if you plan on modifying the OSS code.  If your not modifying it you can use it, even for development.

This is the guidance on using OSS.  These considerations will apply when releasing source code and they are addressed in the other guide.  I thought removing this made it clearer to understand

Reply:
Sounds good, will merge but need a pointer to compliance in publishing.